### PR TITLE
feat/medium

### LINKED ISSUE #70: Add Medium articles section
Connect Thinc's articles in medium and put it in website.

Thinc's Medium: https://medium.com/thinc-org

### PR DESCRIPTION
**Pull request by new5558**
### Demo: https://thinc-www-ihdi0cptz-new17353.vercel.app/ (edited)

image
image

### CODE REVIEW

**Comment by bombnp** [comment]
pls use `fixes #70` or `closes #70` my man

File: .gitignore [hunk 1]
```diff
@@ -38,4 +38,6 @@ jspm_packages
 .next
 
 .DS_Store
-out
\ No newline at end of file
+out
+.vercel
+.vercel
```
Comment (bombnp) [inline .gitignore]: duplicate gitignore?

File: components/sections/medium.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                      src="https://cdn-images-1.medium.com/fit/c/50/50/1*8ddLbmSK4fwLH5bVcO7ITQ.jpeg"
+                      alt="Thinc. medium logo"
+                    />
+                    <a href="https://medium.com/thinc-org" target="_blank" rel="noopener noreferrer">
+                      <h6 className="text-gray-600 text-lg ml-2">Thinc.</h6>
+                    </a>
+                  </div>
+                  <p className="text-gray-600 text-base font-normal leading-relaxed my-4">
+                    Thinc. is a student run organization whose aim is to incubate students for making impact to society
+                    by using IT development.
```
Comment (bombnp) [inline components/sections/medium.jsx]:
```suggestion
                    using IT development.
```

File: components/sections/medium.jsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                  <a href="https://medium.com/thinc-org" target="_blank" rel="noopener noreferrer">
+                    <h6 className="text-yellow-800">More information</h6>
+                  </a>
+                </div>
+              </div>
+              <h6 className="text-2xl self-start font-bold">Publications</h6>
+              <div className="medium-article-container mt-4">{mediumFeed}</div>
+              <a
+                href="https://medium.com/thinc-org"
+                className="inline-block mx-2 mt-6"
```
Comment (bombnp) [inline components/sections/medium.jsx]: increase margin top of all publications button a bit. probably twice as much

File: components/sections/medium.jsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <div className="bg-gray-200 text-gray-600 hover:bg-gray-300 hover:underline px-4 py-3 rounded-lg">
+                  <div className="flex flex-row items-center">
+                    <p className="text-sm">All Publications {'->'}</p>
+                  </div>
+                </div>
```
Comment (bombnp) [inline components/sections/medium.jsx]: should we extract this component? we can reuse it multiple time already

File: pages/index.jsx [hunk 5]
```diff
@@ -24,9 +27,29 @@ const Index = () => (
...
     <Teams />
+    <Medium feeds={mediumFeed} />
     <Members />
     <Footer />
   </>
 )
 
 export default Index
+
+export async function getStaticProps() {
```
Comment (bombnp) [inline pages/index.jsx:39]: does github pages support SSR?

File: components/sections/medium.jsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+            target="_blank"
+            rel="noopener noreferrer"
+            className="border flex medium-article-img bg-gray-700 items-center justify-center text-white text-2xl"
+          >
+            <span>Thinc.</span>
+          </a>
+        )}
+        <a href={feed.link} target="_blank" rel="noopener noreferrer">
+          <h6 className="mt-4 text-2xl font-bold ">{feed.title}</h6>
+        </a>
```
Comment (bombnp) [inline components/sections/medium.jsx:46]: can we also show the first multiple lines of body of article? to know what it's about

**Review by bombnp — CHANGES_REQUESTED** [review]

File: pages/index.jsx (hunk 5 above)
Comment (new5558) [reply pages/index.jsx:39]: It is static site generator not SSR

File: components/sections/medium.jsx (hunk 2 above)
Comment (new5558) [reply components/sections/medium.jsx]: Should we also change this sentence on Thinc. medium website?

File: components/sections/medium.jsx (hunk 6 above)
Comment (new5558) [reply components/sections/medium.jsx:46]: image
I will implement it like this then.

File: pages/index.jsx (hunk 5 above)
Comment (new5558) [reply pages/index.jsx:39]: As we discussed in #70. We may need to run action as a cronjob/action to built a static site from medium API

File: components/sections/medium.jsx (hunk 2 above)
Comment (bombnp) [reply components/sections/medium.jsx]: yes

Comment (bombnp) [reply components/sections/medium.jsx]: `by` is redundant

File: pages/index.jsx (hunk 5 above)
Comment (bombnp) [reply pages/index.jsx:39]: can you open issue for this? so we don't forget to make a cron

File: components/sections/medium.jsx (hunk 6 above)
Comment (new5558) [reply components/sections/medium.jsx:46]: image
Done! here is a the preview

File: components/sections/medium.jsx (hunk 2 above)
Comment (new5558) [reply components/sections/medium.jsx]: I don't have access to Thinc. medium/ can you do it @bombnp?

Comment (bombnp) [reply components/sections/medium.jsx]: I don't. I have no idea who is owner. Even P'Far doesn't i think

File: components/sections/medium.jsx (hunk 6 above)
Comment (bombnp) [reply components/sections/medium.jsx:46]: Neat

**Comment by bombnp** [comment]
Can u redeploy to vercel

**Comment by new5558** [comment]
> Can u redeploy to vercel

Deployed. Check out new deployment on PR description above!

**Comment by bombnp** [comment]
i feel like medium and member section should have different bg color, like the ones in team, project, and about

**Comment by bombnp** [comment]
other than that, LGTM

**Comment by new5558** [comment]
> i feel like medium and member section should have different bg color, like the ones in team, project, and about

After careful selection, I decided to put change the background of the member section to be contrasted with the medium section's.
I also added a gray border in Thinc. medium description as attached image
image
image
image

**Pull request merged**